deploy/klusterlet/olm-catalog/klusterlet/0.1.0/klusterlet.v0.1.0.clusterserviceversion.yaml

```diff
@@ -212,8 +256,16 @@ spec:
     type: AllNamespaces
   keywords:
   - open-cluster-management
+  - Klusterlet
+  links:
+  - name: Klusterlet Operator
```

Comment:
I think I would expect to see `Registration Operator` as the title for this link

Reply:
done